### PR TITLE
Update rooms-lifecycle-support.md

### PR DESCRIPTION
**Pull request by sohailta**

### CODE REVIEW

File: Teams/rooms/rooms-lifecycle-support.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ If you need to, you can delay the upgrade of your Teams Rooms device to Windows
 To delay the upgrade, you need to set the `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\PPI\SkypeSettings\CBU11` value to `1 (DWORD)`. We recommend setting the `CBU11` value using group policy, Intune, or remote PowerShell. Alternately, you can complete the steps below on each Teams Rooms device.
 
 > [!IMPORTANT]
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-lifecycle-support.md:77]:
```suggestion
```

File: Teams/rooms/rooms-lifecycle-support.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ If you need to, you can delay the upgrade of your Teams Rooms device to Windows
...
-> If you want to temporarily delay the Windows 11 upgrade on eligible devices, you need to set the `CBU11` value before June 20, 2023.
+> If you want to temporarily delay the Windows 11 upgrade on eligible devices, you need to set the `CBU11` value before release of 4.17 app (last week of June).
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-lifecycle-support.md:79]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sohailta, kindly commit the suggested changes to remove the important note if you agree with my suggestion. The note should not be relevant if the release date of version 4.17 has passed. Many thanks.

**Comment by scanum** [comment]
We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this issue.  
 
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Pull request closed without merging**